generators/kubernetes/templates/deployment.yaml

```diff
@@ -40,4 +40,36 @@ spec:
           {{#each storageDeploys.env}}
           - name: {{@key}}
             value: {{{this}}}
-          {{/each}}
\ No newline at end of file
+          {{/each}}
+          - name: APPLICATION_NAME
+            value: "{{tag ' .Release.Name '}}"
```

Comment:
Please use `{{{ }}}` so that your `tag ' . Releas Name '` is properly renderede

Reply:
Due to this commit that is no longer required: https://github.com/ibm-developer/generator-ibm-cloud-enablement/commit/29eb3f7438722fa40c223fb8852b3e36506b25b8

Reply:
@katheris right I forgot about that fix. Thanks for the heads up

Reply:
We do need a test to verify that the `{{tag }}` handlebars section is being processed correctly during generation though

Reply:
@Aerex @katheris thanks for help to review this PR.

The "{{tag ' .Release.Name '}}" was discussed and suggested by David Currie. 
 - name: APPLICATION_NAME
   value: "{{tag ' .Release.Name '}}"

And also this have been verified by the java. Here is the similar PR already committed long time ago for java:
https://github.com/ibm-developer/generator-ibm-cloud-enablement/pull/98

Reply:
We need an actual test in the codebase to check the contents after generation, not just a manual verification.

Reply:
Also, rebase your commit afterwards